src/css/base.scss

```diff
@@ -133,3 +133,10 @@ select {
   margin: 10px
 }
 
+.trip {
+  display: flex;
```

Comment:
Bro, do you even flex? (And by that I mean use scss to turn all of your flexin into a mixin/variable/extends to clean up lines of code)

Reply:
I can't confidently answer this question, it's been qqqqquite a while since I've done CSS

Reply:
Mixins are like preschooler functions for scss. They let you assign values with variables to pass into each selected element. @extends are similar, in that they extend all the same properties from the element they are referencing.-i.e
```
button {
background-color: blue;
color: white;
}

.big-button {
@extends button; (will have background-color: blue; and color: white;)
height: 1500px
}
```